### DISCUSSION TITLE
Have a default location and UX for quickly kickstarting new projects

### DISCUSSION THREAD

**Discussion by wmstack** [zed-industries/zed#6771] [Feature Requests]
I believe that editors should allow you to start coding as quickly as possible. In the current world of editors, whenever you are about to start a project, you constantly have to find new places in the file system to put it. You also have to come up with something silly like a name.

There is a UX trend these days within web apps and others of having your app manage your workspaces. For example, in Edstem, you don't have to worry about silly things like where you place your projects in a file system, you just open the web app, create a new workspace put a name, press a button and you can already start coding. In Replit, a random group of words is written by default so you can just have the smoothest project initiation pathway.

It abstracts away all these unnecessary steps. This is similar to a lot of note-taking apps as well, where you don't have to specify where in your filesystem you want to put that unicorn notebook. It causes less friction than things like Obsidian vaults and file explorers. I would rather, for all I care, that the editor vibes like a web app.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
